### PR TITLE
Data Model section editorial review

### PR DESCRIPTION
**Pull request by grantnoble**
Wording and grammar changes,  some punctuation, spacing, line breaks, and line length updates

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/did-core/pull/114.html" title="Last updated on Nov 19, 2019, 11:55 PM UTC (f56bf57)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/114/a8d0c28...grantnoble:f56bf57.html" title="Last updated on Nov 19, 2019, 11:55 PM UTC (f56bf57)">Diff</a>

### CODE REVIEW

**Comment by TallTed** [comment]
There are dueling changes, in this and other PRs, adding and removing "Section" before reference links, e.g., `see Section <a href="#extensibility"></a>` vs `see <a href="#extensibility"></a>`.

The rendered doc includes the `§` symbol which is read as "Section", so I think the correct thing is to *not* include the literal `Section`.

**Comment by grantnoble** [comment]
> There are dueling changes, in this and other PRs, adding and removing "Section" before reference links, e.g., `see Section <a href="#extensibility"></a>` vs `see <a href="#extensibility"></a>`.
> 
> The rendered doc includes the `§` symbol which is read as "Section", so I think the correct thing is to _not_ include the literal `Section`.

To me, the `§` symbol simply reads as a squiggly symbol. The word "Section" says to the reader (who we're doing this for, after all) that the link takes them to another section in the document.

In ReSpec config, there is now an option to remove that symbol. `addSectionLinks: false`

File: index.html [hunk 1]
```diff
@@ -551,104 +551,115 @@ <h1>
...
+The properties that can be present in a <a>DID document</a> are outlined in
+Section <a href="#did-documents"></a>.
+    </p>
+
+    <p>
 The properties present in a <a>DID document</a> can be updated according to
-the applicable <a href="#did-methods"></a>.
-      </p>
-    </section>
+the applicable methods outlined in Section <a href="#did-methods"></a>.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply index.html]: Done!

**Comment by grantnoble** [comment]
Regarding the duelling changes between PRs, do we have a style guide to adjudicate on these sorts of things (whether or not to use the word "Section", (s) to indicate one or more things, etc.).
If not, would it be worthwhile to create one?

**Comment by TallTed** [comment]
I don't care strongly whether we choose [`§`](https://en.wikipedia.org/wiki/Section_sign) or `Section` for use in the body, but it should be one or the other, definitely not both. We should also note that `§` is found in the sidebar throughout the document at each _section_ heading.

As to the singular-or-plural `(s)` -- rewording can remove the need for it in most (possibly all) cases, but that does require more significant care and attention to ensure that the correct meaning is retained, unless, for instance, "controller(s)" is simply replaced by "controller or controllers" (which shows why the shorthand "controller(s)" is used).

**Comment by msporny** [comment]
> I don't care strongly whether we choose the section sign § or Section for use in the body

I'll care strongly about this one... Let's avoid using § and spell out "Section" everywhere. It reduces the reading level of the document ever so slightly *and* some screen readers don't read § out loud, which isn't great for those with accessibility needs.

**Pull request merged**